sdk/tests/integration/TransactionReceiptQueryIntegrationTest.cc

```diff
@@ -83,25 +88,127 @@ class TransactionReceiptQueryIntegrationTest : public ::testing::Test
 
 private:
   Client mClient;
+
+  const AccountId mTestAccountId = AccountId::fromString("0.0.1023");
 };
 
 // Tests invoking of method execute() from TransactionReceiptQuery.
-TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteRequestToTestnetNode)
+TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteAccountCreateTransaction)
+{
+  // Given
+  const auto testPublicKey = ED25519PrivateKey::generatePrivateKey()->getPublicKey();
+
+  // When
+  TransactionReceipt txReceipt;
+  EXPECT_NO_THROW(
+    txReceipt = AccountCreateTransaction().setKey(testPublicKey).execute(getTestClient()).getReceipt(getTestClient()));
+
+  // Then
+  EXPECT_EQ(txReceipt.getStatus(), Status::SUCCESS);
+  EXPECT_TRUE(txReceipt.getAccountId().has_value());
+  EXPECT_TRUE(txReceipt.getExchangeRates().has_value());
```

Comment:
```suggestion
  ASSERT_TRUE(txReceipt.getExchangeRates().has_value());
```

If you plan to test the value, I'd put `ASSERT` instead of `EXPECT`, as failing an `ASSERT` check will cause the test to end, while `EXPECT` will continue. If the `ExchangeRates` doesn't have a value, the next step will crash the program.